website/docs/components/form/file-input/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,33 @@
+## Usage
+
+### When to use
+
+- Use a file input to let users quickly select files from their computer.
+
+### Types of File Inputs 
+
+The appearance of file input can vary across different web browsers.
```

Comment:
I would move this part in a dedicated section in which we explain the visual difference between browsers

Reply:
@andgen404